hazelcast/src/test/java/com/hazelcast/internal/eviction/CacheExpirationManagerTest.java

```diff
@@ -138,6 +142,67 @@ public void stops_running_backgroundClearTask_when_lifecycleState_MERGING() {
         backgroundClearTaskStops_whenLifecycleState(MERGING);
     }
 
+    @Test
+    public void no_expiration_task_starts_on_new_node_after_migration_when_there_is_no_expirable_entry() {
+        Config config = new Config();
+        config.setProperty(taskPeriodSecondsPropName(), "1");
+        TestHazelcastInstanceFactory factory = createHazelcastInstanceFactory();
+        final HazelcastInstance node1 = factory.newHazelcastInstance(config);
+
+        CacheManager cacheManager = createCacheManager(node1);
+        Cache cache = cacheManager.createCache("test", new CacheConfig());
+        cache.put(1, 1);
+
+        final HazelcastInstance node2 = factory.newHazelcastInstance(config);
+        node1.shutdown();
+
+        assertTrueAllTheTime(new AssertTask() {
+            @Override
+            public void run() {
+                assertFalse("There should be zero CacheClearExpiredRecordsTask",
+                        hasClearExpiredRecordsTaskStarted(node2));
+            }
+        }, 3);
+    }
+
+    @Test
+    public void expiration_task_starts_on_new_node_after_migration_when_there_is_expirable_entry() {
+        Config config = new Config();
+        config.setProperty(taskPeriodSecondsPropName(), "1");
+        TestHazelcastInstanceFactory factory = createHazelcastInstanceFactory();
+        final HazelcastInstance node1 = factory.newHazelcastInstance(config);
+
+        CacheManager cacheManager = createCacheManager(node1);
+        Cache cache = cacheManager.createCache("test", new CacheConfig());
+        ((ICache) cache).put(1, 1, new ExpiryPolicy() {
```

Comment:
you can use `new HazelcastExpiryPolicy(10000, 10000, 10000)` to make this shorter.

Reply:
done.